Go/{{.ProjectDirectory}}/go.mod

```diff
@@ -0,0 +1,10 @@
+module {{ .ModuleName }}
+
+go 1.18
+
+require (
+	github.com/BurntSushi/toml v1.1.0 // indirect
+	github.com/Masterminds/semver/v3 v3.1.1 // indirect
+	github.com/buildpacks/libcnb v1.26.0 // indirect
+	github.com/onsi/gomega v1.19.0 // indirect
+)
```

Comment:
I wonder if dependabot will keep these up to date or not? 🤔

Regardless, we may want the template to create a README.md or TODO.md in the generated buildpack directory with some next steps for the user. Updating go modules could be one thing. Modify build and detect or create a new module and tests for specific functionality.

Reply:
I've added a `README.md`.  I don't think dependabot will keep our dependencies up-to-date as the `go.mod` file does not parse.  It does not parse due to the `module {{ .ModuleName }}` template variable.